packages/next/test/unit/fixtures/01-normalize-routes/yarn.lock

```diff
@@ -1,184 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Lets keep the lock file because it should be faster, right?

Reply:
I can't speak to the performance, but the `package.json` has all deps as `"latest"` and since that can change, creating a lock file for a single point in time may cause issues down the road. On the other hand, 6 of the 90-some next builder fixtures have lock files. I really don't think we need to weigh down the repo with this file, but I'm indifferent.

Reply:
@cb1kenobi This test is not really testing Next.js though, its testing the Builder so I would prefer it be faster

Reply:
Good idea! Here's the PR to add the lock file back: https://github.com/vercel/vercel/pull/8600.